manifests/stage-ib-kubernetes/0070-deployment.yaml

```diff
@@ -71,6 +71,15 @@ spec:
                     values:
                       - ib-kubernetes
               topologyKey: kubernetes.io/hostname
+      tolerations:
+        - key: "node-role.kubernetes.io/master"
+          operator: "Equal"
```

Comment:
shouldnt it be operator "Exist" ?

did you test it ? does this form work ?

Reply:
https://github.com/Mellanox/network-operator/blob/baa3be5fb7719f428cb42fb920fe990ab27b729d/manifests/stage-ofed-driver/0050_ofed-driver-ds.yaml#L36

Reply:
You are right, thank you. Applied the change